k8s/k8s_test.go

```diff
@@ -14,7 +19,27 @@ func Test_ParseVulneDB(t *testing.T) {
 	var bi CVE
 	err = json.Unmarshal(b, &bi)
 	assert.NoError(t, err)
-	kvd, err := ParseVulnDBData(bi, map[string]string{})
+
+	mux := http.NewServeMux()
+	err = filepath.WalkDir("./testdata/mitreCVEs", func(path string, d fs.DirEntry, err error) error {
+		if !d.IsDir() {
+			var f []byte
+			f, err = os.ReadFile(path)
+			assert.NoError(t, err)
+			mux.HandleFunc(fmt.Sprintf("/%s", filepath.Base(path)), func(w http.ResponseWriter, _ *http.Request) {
+				_, err = w.Write(f)
+				assert.NoError(t, err)
+			})
```

Comment:
We might be able to leverage `http.FileServer`.
https://pkg.go.dev/net/http#FileServer

If you need to modify the URL's path, `http.StripPrefix` can help.
https://pkg.go.dev/net/http#StripPrefix

Reply:
Thanks for the advice!